From 8cdf846922ffb3e0d2a828be473fdf9c45a56e7c Mon Sep 17 00:00:00 2001
From: Rosen Penev <rosenp@gmail.com>
Date: Sat, 30 Mar 2019 20:19:17 -0700
Subject: [PATCH] intel_cpus.cpp: Change open parameter to const char *

ifstream::open takes std::string starting with C++11, not before.

This fixes compilation with uClibc++ and potentially other older libraries

Signed-off-by: Rosen Penev <rosenp@gmail.com>
---
 src/cpu/intel_cpus.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/cpu/intel_cpus.cpp
+++ b/src/cpu/intel_cpus.cpp
@@ -92,7 +92,7 @@ int is_supported_intel_cpu(int model, in
 
 int is_intel_pstate_driver_loaded()
 {
-	const string filename("/sys/devices/system/cpu/cpu0/cpufreq/scaling_driver");
+	const char *filename = "/sys/devices/system/cpu/cpu0/cpufreq/scaling_driver";
 	const string intel_pstate("intel_pstate");
 	char line[32] = { '\0' };
 	ifstream file;
